src/main/java/com/fryrank/model/Address.java

```diff
@@ -0,0 +1,21 @@
+package com.fryrank.model;
+
+import lombok.Data;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+@Data
+public class Address {
```

Comment:
We should include country as a field here, so that we can easily add other countries in later.

Reply:
I was wondering about that since addresses differ from country to country. For now I was planning on going with US addresses for simplicity (and since our data will probably be wiped before the final iteration of our schema), but I think if we want to add in a backdoor for other countries in the future, we may want to just go very generic with the address class i.e.

```java 
String countryISOCode;
// City, municipality, town, geographical area
String locality;
// Administrative division, State, Province, Prefecture
String region; 
String postalCode;
String streetAddress;
```

How does that sound?

Reply:
Ok, why don't we leave it as you originally had it. We can limit our scope to the US for now and expand that to worldwide when we leverage some API like Google's to get more restaurants.

Reply:
Limiting the scope 👍